docs/README.md

```diff
@@ -97,6 +97,18 @@ Run `bundle update` to make sure you're using the most recent Ruby gem versions.
 
 Run `bundle exec middleman build --verbose` to get detailed error messages to help with finding the problem.
 
+## Diagrams
+
+Diagrams are maintained in [DrawIO](https://app.diagrams.net) using the file [monitoring_as_code.drawio](https://github.com/HO-CTO/sre-monitoring-as-code/blob/369-render-diagrams/docs/monitoring_as_code.drawio). 
+Once landing in DrawIO web interface select `Save diagrams to: Device > Open Existing Diagram` and select [monitoring_as_code.drawio](https://github.com/HO-CTO/sre-monitoring-as-code/blob/369-render-diagrams/docs/monitoring_as_code.drawio) from you're local repository. 
```

Comment:
Just a small nit pick, I think these links will break as soon as the branch is deleted.

Reply:
My mistake, good spot. Thanks Sam